src/extension.ts

```diff
@@ -218,3 +226,38 @@ export function activate(context: vscode.ExtensionContext) {
 
 export function deactivate() {
 }
+
+function activateDps(context: vscode.ExtensionContext) {
+    let packageJSON = vscode.extensions.getExtension(Constants.ExtensionId).packageJSON;
+    let telemetryReporter = new TelemetryReporterWrapper(Constants.ExtensionId, packageJSON.version, packageJSON.aiKey, context, "DPS.");
```

Comment:
Can we use telemetryClient? If it required telemetryReporter, we could make telemetryClient decorator pattern.